Include/unicodeobject.h

```diff
@@ -97,6 +97,11 @@ Copyright (c) Corporation for National Research Initiatives.
 #  include <wchar.h>
 #endif
 
+#ifdef HAVE_NON_UNICODE_WCHAR_T_REPRESENTATION
+#  include <uchar.h>
+#  include <langinfo.h>
+#endif
```

Comment:
Why do you add includes here?

Reply:
It's necessary for the `char32_t` type.

Reply:
unicodeobject.h is included by Python.h, but char32_t is only needed by Include/cpython/fileutils.h, no?

Reply:
`char32_t` is no longer used and hence the import gone.